server/server.go

```diff
@@ -241,6 +245,28 @@ func (server HTTPServer) proxyTo(baseURL string) func(http.ResponseWriter, *http
 	}
 }
 
+// getGroups retrives the groups configuration from a channel to get the latest valid one and send the response back to the client
+func (server *HTTPServer) getGroups(writer http.ResponseWriter, request *http.Request) {
+	select {
```

Comment:
What's the purpose of this `select`? It's reading only from one channel, isn't it?

Reply:
You are right, I don't remember why I put it, but it is not needed at all. TYVM!

Reply:
Are you ok with the comments included in the last version?

Reply:
Fixed in the last commit